webapp/graphite/render/functions.py

```diff
@@ -2958,6 +2957,33 @@ def reduceSeries(requestContext, seriesLists, reduceFunction, reduceNode, *reduc
     metaSeries[key].name = key
   return [ metaSeries[key] for key in keys ]
 
+def applyByNode(requestContext, seriesList, nodeNum, templateFunction, newName=None):
+  """
+  Takes a seriesList and applies some complicated function (described by a string), replacing templates with uniquified
+  values from the seriesList up to some unique node.
+
+  ...code-block:: none
+
+      &target=applyByNode(servers.*.disk.bytes_used,1,"divideSeries(%.disk.bytes_used, sumSeries(%.disk.bytes_*)))
```

Comment:
Looks like you are opening a double-quote at `divideSeries` that does not get closed?